.jenkins/Jenkinsfile

```diff
@@ -220,7 +220,23 @@ def win2016CrossCompile(String build_type, String has_quote_provider = 'OFF') {
                       vcvars64.bat x64 && \
                       cmake.exe ${WORKSPACE} -G Ninja -DCMAKE_BUILD_TYPE=${build_type} -DBUILD_ENCLAVES=ON -DHAS_QUOTE_PROVIDER=${has_quote_provider} -DNUGET_PACKAGE_PATH=C:/oe_prereqs -Wdev && \
                       ninja.exe && \
-                      ctest.exe -V -C ${build_type} --timeout ${CTEST_TIMEOUT_SECONDS}
+                      ctest.exe -V -C ${build_type} --timeout ${CTEST_TIMEOUT_SECONDS} && \
+                      cpack.exe -D CPACK_NUGET_COMPONENT_INSTALL=ON -DCPACK_COMPONENTS_ALL=OEHOSTVERIFY && \
+                      cpack.exe && \
+                      rmdir /s/q C:/oe && \
+                      nuget.exe install open-enclave -Source %cd% -OutputDirectory C:\\oe -ExcludeVersion && \
+                      set OpenEnclave_DIR=C:\\oe\\open-enclave\\openenclave\\lib\\openenclave\\cmake && \
+                      cd C:\\oe\\open-enclave\\openenclave\\share\\openenclave\\samples && \
+                      setlocal enabledelayedexpansion && \
+                      for /d %%i in (*) do (
+                          cd "%%i"
```

Comment:
Because result of "cd ..\\.." of last step is out of control of this script, it may be more reliable to use absolute path here as 

> cd C:\\oe\\open-enclave\\openenclave\\share\\openenclave\\samples\\"%%i"

Reply:
Why is "cd ../.." out of control of this script?

Reply:
cmake, ninja above "cd ..\.." may change to other dir and not stay in build, since cmake and ninja use other files as input.

Reply:
Okay, I got it.

Reply:
Just to clarify, will cmake or ninja change to other directory under successful calls? Because if cmake or ninja hit an error, the script will return immediately.

Reply:
Any call exit value equals 0 will be treated as a successful call. If the input of ninja is just "cd ..\\", after execution the exit value is also 0. Of course this case is extremely abnormal. Since someone will add samples in the future,  it is better to rely on this script rather than other's input.

Reply:
Thanks for the clarification! I will change this.